### ISSUE TITLE
Could the function adjust the distance between the setname and the circle?

### ISSUE THREAD

**Issue by SSSJe** [yanlinlin82/ggvenn#40]
I used the function:
`ggvenn(venn_list, fill_color = c("#1F77B4FF", "#FF7F0EFF"), fill_alpha = .7, stroke_size = 0.3, stroke_alpha = .5, show_percentage = FALSE, text_size = 8, set_name_size = 6) + 
labs(title = 'TNK') + 
theme(plot.title = element_text(size=18, face="bold", hjust=0.5))`
The output is like this:
![image](https://github.com/yanlinlin82/ggvenn/assets/62241145/841ce5af-8f00-49bb-bdb2-7d5c96eb114b)
 I want to adjust the distance between the setnames (such as HCC and ICC) and the circle, or do something which making the title (TNK) will not cover the setnames.

**Comment by yanlinlin82** [comment]
Hi, @SSSJe 

Will you try this please:

```r
g = ggvenn(...) + ... # your venn plotting

# Adjusting plot margin and title space
g + theme(
  plot.margin = margin(2, 2, 2, 2, "cm"), # Increase plot margins (top, right, bottom, left)
  plot.title = element_text(size = 20, hjust = 0.5, vjust = -1,
    margin = margin(b = 1, unit = "cm")), # Increase title size and space below the title
)
```

**Comment by SSSJe** [comment]
> Hi, @SSSJe
> 
> Will you try this please:
> 
> ```r
> g = ggvenn(...) + ... # your venn plotting
> 
> # Adjusting plot margin and title space
> g + theme(
>   plot.margin = margin(2, 2, 2, 2, "cm"), # Increase plot margins (top, right, bottom, left)
>   plot.title = element_text(size = 20, hjust = 0.5, vjust = -1,
>     margin = margin(b = 1, unit = "cm")), # Increase title size and space below the title
> )
> ```
Thank you very much!!! Lol
It's useful to solve  this problem~

**Comment by SSSJe** [comment]
amazing… With my trying, the current situation doesn't require setting edge values, and there won't be any occlusion according to my previous method.

**Comment by dnldelarosa** [comment]
This issue has been resolved by the following pull request: #47 

The new `padding` argument in the `ggvenn` function allows users to control the padding around the Venn diagram, which addresses the issue of truncated labels.

Thank you for reporting this issue!
